config.js

```diff
@@ -0,0 +1,6 @@
+const DB_HOST =
+  "mongodb+srv://Maksym:gsX6EJ8lZu51ccLM@cluster0.ympu5fp.mongodb.net/db-contacts?retryWrites=true&w=majority";
```

Comment:
this string must be inside of dotenv. Should be invisible for everyone

Reply:
corrected